cmd/api/main.go

```diff
@@ -0,0 +1,127 @@
+package main
```

Comment:
the main package needs to be in the root folder

Reply:
Examples of industry leading projects with the same design pattern

https://github.com/heptio/ark/tree/master/cmd (just a really small main function with everything else in packages)
https://github.com/moby/moby/tree/master/cmd
https://github.com/prometheus/prometheus/tree/master/cmd
https://github.com/influxdata/influxdb/tree/master/cmd
https://github.com/satellity/satellity/tree/master/cmd/satellity
https://github.com/dapr/dapr/tree/master/cmd